src/components/ProfileModal/EditProfileModal.tsx

```diff
@@ -72,6 +72,7 @@ function EditProfileModal(props: EditProps) {
             _id: props.userData?._id!,
           });
           setInvalidEmail("");
+          window.location.reload();
```

Comment:
Is there a way to do this with `useState()` or `useEffect()`? Reloading the entire page is usually not necessary.